build/centos7-puppet6/Puppetfile

```diff
@@ -55,7 +56,11 @@ mod 'puppet-nginx'
 mod 'puppetlabs-concat'    # dependency of puppet-nginx'
 mod 'puppet-nodejs'
 mod 'puppet-python'
-mod 'puppet-rabbitmq'
+# mod 'puppet-rabbitmq'
+mod 'puppet/rabbitmq',
+    :git => 'https://github.com/EncoreTechnologies/puppet-rabbitmq.git',
+    :branch => 'feature/fix-yum-repo',
+    :default_branch => 'master'
```

Comment:
Can we have a small code comment reminder about the upstream change that's waiting to be merged?
So we could switch to upstream again in the future.